.github/workflows/content.yml

```diff
@@ -0,0 +1,133 @@
+name: Release Production
+
+on:
+    push:
+        tags:
+            - 'production*'
+        paths-ignore:
+            - '**.md'
```

Comment:
we can not allow tag based push operation for this new workflow. then it would be same as production. since this action is going to start by some events then we need to use repository dispatch. link to documentation: https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#repository_dispatch